pyproject.toml

```diff
@@ -115,6 +115,12 @@ filterwarnings = [
     "module:unclosed <socket.socket:ResourceWarning",
     "module:unclosed <ssl.SSLSocket:ResourceWarning",
     "module:unclosed <socket object:ResourceWarning",
+    # TODO: Remove the next five as part of PYTHON-5036.
```

Comment:
What do these resource warning have to do with unittest? They seem like unrelated resource leaks in the test.

Reply:
Ping.